cluster-scope/base/core/namespaces/thoth-pulp-experiments/namespace.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/display-name: "Pulp+Thoth-Station: Experimenting with Pulp"
+    openshift.io/requester: thoth-pulp-experiments
```

Comment:
Can we please match the requester annotation to a single OCP user group that owns this namespace?

Reply:
yes definitely , thank you.